docs/dynamic-rupture.rst

```diff
@@ -365,3 +363,31 @@ Two additional thermal pressurization parameters are space-dependent and therefo
     tp_halfWidthShearZone: 0.01     # Half width of shearing zone [m]
 
 TP generates 2 additional on-fault outputs: Pore pressure and temperature (see fault output).
+
+
+Nucleation
+~~~~~~~~~~~~~~~~~~~~~~
+
+SeisSol provides several options to enforce artificial nucleation within a limited temporal and spatial extent.
+The additional nucleation stress applied to the fault plane can be specified using the parameters ``Tnuc_s`` (along strike), ``Tnuc_d`` (along dip), and ``Tnuc_n`` (normal stress).
+Alternatively, if the stress state is described by the full stress tensor, the parameters ``nuc_xx``, ``nuc_yy``, ``nuc_zz``, ``nuc_yz``, ``nuc_xz``, and ``nuc_xy`` can be used.
```

Comment:
"the parameters [...] can be used" -> "use the parameters [...]"
(since you'll need to do that)